### ISSUE TITLE
Fails on Haml > 6.3

### ISSUE THREAD

**Issue by damisul** [sds/haml-lint#619]
We've updated our project to HAML 7, but it seems haml_lint does not support it yet, it fails with a message:
```
Cannot handle Haml version: ~> 7.0
```

I wonder if there are any plans to support it?

**Comment by sds** [comment]
Update to HAML-Lint [0.67.0](https://github.com/sds/haml-lint/releases/tag/v0.67.0) or newer.

**Comment by damisul** [comment]
Oh, thank you. Sorry for false report. I've tried to update haml_lint before reporting, but it seems there is some issues in our Gemfile and it stayed on old version
